sdks/java/core/src/test/java/com/google/cloud/dataflow/sdk/testing/TestPipelineTest.java

```diff
@@ -91,4 +91,18 @@ public void testCreationOfPipelineOptionsFromReallyVerboselyNamedTestCase() thro
   public void testToString() {
     assertEquals("TestPipeline#TestPipelineTest-testToString", TestPipeline.create().toString());
   }
+
+  @Test
+  public void testToStringNestedMethod() {
```

Comment:
Add a test for the nested class case:
TestClass.myTest
TestClass.MyTestClassHelper.runPipeline
TestPipeline.run